src/components/Hearing.js

```diff
@@ -27,14 +27,14 @@ class Hearing extends React.Component {
     const hearingId = this.props.hearingId;
     const {authCode} = this.props.location.query;
     const mainSection = find(this.props.hearing.sections, (section) => section.type === "main");
-    dispatch(postSectionComment(hearingId, mainSection.id, text, authorName, authCode));
+    dispatch(postSectionComment(hearingId, mainSection.id, text, authorName, null, authCode, null, null, null));
   }
 
-  onPostSectionComment(sectionId, text, authorName, pluginData) {
+  onPostSectionComment(sectionId, text, authorName, pluginData, geojson, label, images) {
```

Comment:
Likewise, this should probably also be refactored to less indexed args and more structured ones.